content/en/blog/2023/native-sidecars/index.md

```diff
@@ -0,0 +1,207 @@
+The sidecar pattern is just that: a pattern.
+Up until this point, there has been no formal support for 'sidecar containers' in Kubernetes at all.
+
+This has caused no end of problems: what if you have a job that terminates by design, but a sidecar container that doesn't?
+This exact use case is the [most popular ever on the Kubernetes issue tracker](https://github.com/kubernetes/kubernetes/issues/25908).
+
+A formal proposal for adding sidecar support in Kubernetes was raised in 2019, with many stops and starts along the way.
+After a reboot of the project last year, formal support for sidecars is being released to Alpha in Kubernetes 1.28.
+Istio has implemented support for this feature, and in this post you can learn how to take advantage of it.
+
+## Sidecar woes
+
+Sidecar containers give a lot of power, but come with some issues.
+While containers within a Pod can share some things, their *lifecycle's* are entirely decoupled.
+To Kubernetes, both of these containers are functionally the same.
+
+However, in Istio they are not the same - the Istio container is required for the primary application container to run,
+and has no value without the primary application container.
+
+This mismatch in expectation leads to a variety of issues:
+* If the application container starts faster than Istio's container, it cannot access the network.
+  This wins the [most +1's](https://github.com/istio/istio/issues/11130) on GitHub by a landslide.
+* If Istio's container shuts down before the application container, it cannot access the network.
+  This often means we shutdown too fast (breaking the network) or too slow (leaving pods running with no purpose).
+* If an application container intentionally exits (typically from usage in a `Job`), Istio's container will still run and keep the pod running indefinitely.
+  This is also a [top GitHub issue](https://github.com/istio/istio/issues/11659).
+* `InitContainers`, which run before Istio's container starts, cannot access the network.
+
+Countless hours have been spent in the Istio community and beyond to work around these - to limited success.
+
+## Fixing the root cause
+
+While increasingly-complex workarounds in Istio can help alleviate the pain for Istio users, ideally all of this would just work - and not just for Istio.
+Fortunately, the Kubernetes community has been hard at work to address these directly in Kubernetes.
+
+In Kubernetes 1.28, a new feature to add native support for sidecars was merged, closing out over 5 years of ongoing work.
+With this merged, all of our issues can be addressed without workarounds!
+
+While we are on the "GitHub issue hall of fame", [these](https://github.com/kubernetes/kubernetes/issues/25908) [issues](https://github.com/kubernetes/kubernetes/issues/65502) account for #1 and #6 all time issues in Kubernetes - and have finally been closed!
```

Comment:
I'm not sure of a better way to write this, but looking at in initially I was wondering where the second issue was (it looks like a single link). Maybe add the word two in between:
```suggestion
While we are on the "GitHub issue hall of fame", [these](https://github.com/kubernetes/kubernetes/issues/25908) two [issues](https://github.com/kubernetes/kubernetes/issues/65502) account for #1 and #6 all time issues in Kubernetes - and have finally been closed!
```